### PR TITLE
fix bug 1115767 - SMTP headers for url & username

### PR DESCRIPTION
**Pull request by groovecoder**
https://bugzilla.mozilla.org/show_bug.cgi?id=1115767

Spot-check instructions are the same as https://github.com/mozilla/kuma/pull/2966, but check for the custom SMTP headers too.

I'm not super-happy with this code so I could use a critical review on it.

### CODE REVIEW

**Comment by groovecoder** [comment]
Either @ubernostrum or @jezdez should tell me a better way to do this ...

File: kuma/wiki/views.py [hunk 1]
```diff
@@ -2186,7 +2187,11 @@ def _make_first_edit_email(new_rev, request):
     template = 'wiki/email/edited.ltxt'
     context = context_dict(new_rev)
     message = render_to_string(request, template, context)
-    email = EmailMessage(subject, message, settings.DEFAULT_FROM_EMAIL)
+    article_url = "%s%s" % (Site.objects.get_current().domain,
+                            doc.get_absolute_url())
+    email = EmailMessage(subject, message, settings.DEFAULT_FROM_EMAIL,
+                         headers = {'X-Article-Url': article_url,
+                                    'X-Editor-Username': user.username})
```
Comment (jezdez) [inline kuma/wiki/views.py]: While RFC 822 is not mentioning it, it's good practice to let the extension headers contain the product they are from, so let's use `X-Kuma-Document-URL` and `X-Kuma-Document-Editor`.

File: kuma/wiki/views.py [hunk 2]
```diff
@@ -2186,7 +2187,11 @@ def _make_first_edit_email(new_rev, request):
     template = 'wiki/email/edited.ltxt'
     context = context_dict(new_rev)
     message = render_to_string(request, template, context)
-    email = EmailMessage(subject, message, settings.DEFAULT_FROM_EMAIL)
+    article_url = "%s%s" % (Site.objects.get_current().domain,
+                            doc.get_absolute_url())
+    email = EmailMessage(subject, message, settings.DEFAULT_FROM_EMAIL,
+                         headers = {'X-Article-Url': article_url,
```
Comment (jezdez) [inline kuma/wiki/views.py]: No spaces around `=`

**Comment by jezdez** [comment]
Other than the comments above, r+.

**Comment by groovecoder** [comment]
Updated.

**Comment by jezdez** [comment]
@groovecoder My point was to use Kuma since that's the software, and MDN the platform implemented with it. The head should have Kuma in there as that's how headers behave usually.

Also please mind my other proposal to use "document" instead of "article" in the header name since that's more in line what the wiki app in Kuma contains.

r-

**Comment by groovecoder** [comment]
Updated again.

**Pull request merged**